### ISSUE TITLE
Upgrade Yarn dev dependencies

### ISSUE THREAD

**Issue by markphelps** [markphelps/flipt#112]
Some dev dependencies are several versions behind. Would like to get them as up to date as possible.

```shell
~/workspace/flipt/ui master
❯ yarn outdated
yarn outdated v1.17.3
info Color legend :
 "<red>"    : Major Update backward-incompatible updates
 "<yellow>" : Minor Update backward-compatible features
 "<green>"  : Patch Update backward-compatible bug fixes
Package                            Current Wanted  Latest Package Type    URL
babel-loader                       7.1.5   7.1.5   8.0.6  devDependencies https://github.com/babel/babel-loader
copy-webpack-plugin                4.6.0   4.6.0   5.0.3  devDependencies https://github.com/webpack-contrib/copy-webpack-plugin
css-loader                         0.28.11 0.28.11 3.0.0  devDependencies https://github.com/webpack-contrib/css-loader
eslint                             5.16.0  5.16.0  6.0.1  devDependencies https://eslint.org
eslint-config-prettier             3.6.0   3.6.0   6.0.0  devDependencies https://github.com/prettier/eslint-config-prettier#readme
eslint-config-standard             10.2.1  10.2.1  13.0.1 devDependencies https://github.com/standard/eslint-config-standard
eslint-friendly-formatter          3.0.0   3.0.0   4.0.1  devDependencies https://github.com/royriojas/eslint-friendly-formatter#readme
eslint-loader                      1.9.0   1.9.0   2.2.1  devDependencies https://github.com/webpack-contrib/eslint-loader#readme
eslint-plugin-node                 5.2.1   5.2.1   9.1.0  devDependencies https://github.com/mysticatea/eslint-plugin-node#readme
eslint-plugin-promise              3.8.0   3.8.0   4.2.1  devDependencies https://github.com/xjamundx/eslint-plugin-promise#readme
eslint-plugin-standard             3.1.0   3.1.0   4.0.0  devDependencies https://github.com/standard/eslint-plugin-standard#readme
file-loader                        1.1.11  1.1.11  4.0.0  devDependencies https://github.com/webpack-contrib/file-loader
html-webpack-plugin                2.30.1  2.30.1  3.2.0  devDependencies https://github.com/jantimon/html-webpack-plugin
optimize-css-assets-webpack-plugin 3.2.1   3.2.1   5.0.3  devDependencies http://github.com/NMFR/optimize-css-assets-webpack-plugin
ora                                1.4.0   1.4.0   3.4.0  devDependencies https://github.com/sindresorhus/ora#readme
portfinder                         1.0.20  1.0.21  1.0.21 devDependencies https://github.com/indexzero/node-portfinder#readme
postcss-import                     11.1.0  11.1.0  12.0.1 devDependencies https://github.com/postcss/postcss-import#readme
postcss-loader                     2.1.6   2.1.6   3.0.0  devDependencies https://github.com/postcss/postcss-loader#readme
postcss-url                        7.3.2   7.3.2   8.0.0  devDependencies https://github.com/postcss/postcss-url#readme
semver                             5.7.0   5.7.0   6.2.0  devDependencies https://github.com/npm/node-semver#readme
shelljs                            0.7.8   0.7.8   0.8.3  devDependencies http://github.com/shelljs/shelljs
uglifyjs-webpack-plugin            1.3.0   1.3.0   2.1.3  devDependencies https://github.com/webpack-contrib/uglifyjs-webpack-plugin
url-loader                         0.5.9   0.5.9   2.0.1  devDependencies https://github.com/webpack-contrib/url-loader
vue-loader                         13.7.3  13.7.3  15.7.0 devDependencies https://github.com/vuejs/vue-loader
vue-style-loader                   3.1.2   3.1.2   4.1.2  devDependencies https://github.com/vuejs/vue-style-loader#readme
webpack                            3.12.0  3.12.0  4.36.1 devDependencies https://github.com/webpack/webpack
webpack-dev-server                 2.11.5  2.11.5  3.7.2  devDependencies https://github.com/webpack/webpack-dev-server#readme
   Done in 1.07s.
```

**Comment by markphelps** [comment]
@christopherdiehl

**Comment by christopherdiehl** [comment]
@markphelps please see #113

**Comment by markphelps** [comment]
Fixed by #113
